### ISSUE TITLE
CMake and CTest for fortran track

### ISSUE THREAD

**Issue by pclausen** [exercism/fortran#29]
So I have now setup a branch fortran-cmake in fork:
https://github.com/pclausen/fortran/tree/fortran-cmake

The main CMakelist.txt is a modified version of the one from https://github.com/exercism/cpp

First build only includes hello-world and kind of completes OK
https://travis-ci.org/pclausen/fortran/jobs/418407246

But I get an error in `The command "bin/configlet lint ." exited with 1.` Do we need this configlet lint ?

The next step is to convert leap or bob to use ctest and some asserts which I am thinking a bit about. When it gets a bit further I will create a pull request(?)

I also updated installation doc https://github.com/pclausen/fortran/blob/fortran-cmake/docs/INSTALLATION.md 

Suggestions for improvements are very welcome. I dont know if you can see or commit to my branch. let me know if you need access and please also write how I can give you access.

**Comment by kytrinyx** [comment]
We do need the configlet lint, as it ensures that the Track configuration will integrate properly with the website. The complaint is:

``
-> The implementation for 'hello-world' is missing a test suite.
```

So it looks like it doesn't know how to recognize the test suite.

To fix it you'll need to change the `config.json#/test_pattern` value to recognize the new cmake tests.

You should be able to run the configlet locally if you get the right one for your system here:
https://github.com/exercism/configlet/releases/latest

> I dont know if you can see or commit to my branch. 

We can see it, but not commit to it (we shouldn't need to commit to it, though, so we're probably fine).

If you want to make a "work in progress" pull request, that will give people a place to comment in-line in your changes. To do so, open up your branch to https://github.com/pclausen/fortran/tree/fortran-cmake and then look for the "new pull request button":

<img width="352" alt="screen shot 2018-08-21 at 1 36 49 pm" src="https://user-images.githubusercontent.com/276834/44424768-4c875a00-a547-11e8-8d98-3917b469721c.png">

**Comment by pclausen** [comment]
I created a pull request but it is my first so let me know if something is wrong or incomplete.

**Comment by kotp** [comment]
You can mention the PR like `exercism/fortran#39` which will show up as exercism/fortran#39 and will link the issues and the PR's a bit more directly, but providing a "mentioned in ..." message.  Issues can also be closed automatically by providing a `closes: #whatever_issue_number`  message in a commit.

**Comment by pclausen** [comment]
This was closed as part of Pull requests #36-#62.
